middleware/dnssec/setup.go

```diff
@@ -49,47 +52,57 @@ func dnssecParse(c *caddy.Controller) ([]string, []*DNSKEY, error) {
 			}
 
 			for c.NextBlock() {
-				k, e := keyParse(c)
-				if e != nil {
-					return nil, nil, e
+				switch c.Val() {
+				case "key":
+					k, e := keyParse(c)
+					if e != nil {
+						return nil, nil, 0, e
+					}
+					keys = append(keys, k...)
+				case "cache_capacity":
+					if !c.NextArg() {
+						return nil, nil, 0, c.ArgErr()
+					}
+					value := c.Val()
+					cacheCap, err := strconv.Atoi(value)
```

Comment:
Could you fix this (or at least specify a minimum). Then is ready to be merged (and we can release 001 :-) )

Thanks!